apps/editing-toolkit/editing-toolkit-plugin/editor-site-launch/src/launch-steps/plan-step/index.tsx

```diff
@@ -48,8 +48,13 @@ const PlanStep: React.FunctionComponent< LaunchStepProps > = ( { onPrevStep, onN
 		<LaunchStepContainer>
 			<div className="nux-launch-step__header">
 				<div>
-					<Title>{ __( 'Select a plan', 'full-site-editing' ) }</Title>
-					<SubTitle>
+					<Title data-e2e-string="Select a plan">
+						{ __( 'Select a plan', 'full-site-editing' ) }
+					</Title>
+					<SubTitle
+						data-e2e-string="Pick a plan that’s right for you. Switch plans as your needs change. There’s no risk, you can cancel for a full refund within %1$d days."
+						data-e2e-string-params="[14]"
+					>
```

Comment:
Same as https://github.com/Automattic/wp-calypso/pull/51000/files#r593014779

Reply:
Moved the `data-e2e-*` attributes to `client/landing/gutenboarding/onboarding-block/plans/index.tsx` in https://github.com/Automattic/wp-calypso/pull/51000/commits/14a407eb5c6c941851d5a92e8d20ac1c89e97c3a.